x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -284,6 +284,7 @@ const getSavedObjectTypes = (
               type: 'nested',
               properties: {
                 synthetic_source: { type: 'boolean' },
+                tsdb: { type: 'boolean' },
```

Comment:
@elastic/kibana-core hey, could you approve the change in core? It is due to a new property added to a fleet saved object.